modules/optimeraRtdProvider.js

```diff
@@ -0,0 +1,183 @@
+/**
+ * This module adds optimera provider to the real time data module
+ * The {@link module:modules/realTimeData} module is required
+ *
+ * The module will fetch targeting values from the Optimera server
+ * and apply the tageting to each ad request. These values are created
+ * from the Optimera Mesaurement script which is installed on the
+ * Publisher's site.
+ *
+ * @module modules/optimeraRtdProvider
+ * @requires module:modules/realTimeData
+ */
+
+/**
+ * @typedef {Object} ModuleParams
+ * @property {string} clientID
+ * @property {string} optimeraKeyName
+ * @property {string} device
+ */
+
+import * as utils from '../src/utils.js';
+import { submodule } from '../src/hook.js';
+import { ajaxBuilder } from '../src/ajax.js';
+
+/** @type {ModuleParams} */
+let _moduleParams = {};
+
+/**
+ * Default Optimera Key Name
+ * This can default to hb_deal_optimera for publishers
+ * who used the previous Optimera Bidder Adapter.
+ * @type {string} */
+export let optimeraKeyName = 'hb_deal_optimera';
+
+/**
+ * Optimera Score File Base URL.
+ * @type {string}
+ */
+export const scoresBaseURL = 'https://dyv1bugovvq1g.cloudfront.net/';
```

Comment:
please make this configurbale in case a publisher wants to host the js

Reply:
This commend will address this and the next three comments.

The js file is actually our data endpoint, so the publisher cannot host this file. They have to fetch it on every page load, and it contains the targeting data for the page.

I have added comments to the variable and the setScoresURL() function to help clear up any confusion that this is a library.